ComposableArchitecture/App/Sources/Actions.swift

```diff
@@ -0,0 +1,29 @@
+enum AppAction {
+    case home(HomeAction)
+    case recents(RecentsAction)
+    case favorites(FavoritesAction)
+    case word(WordAction)
+}
```

Comment:
Should we update swift-format to use 4 spaces? (The default is 2)

Reply:
I suppose so, Xcode defaults to 4 IIRC